server/executor/runner.go

```diff
@@ -159,6 +158,13 @@ func (r persistentRunner) processExecQueue(job execReq) {
 		panic(err)
 	}
 
+	if job.test.ServiceUnderTest.Type == model.TriggerTypeTRACEID {
```

Comment:
So you either have to choose if you want an HTTP test or a test that you provide the trace?

I remember some discussions that we had in the past where we considered it would be nice to be able to provide a trace ID for any kind of test. This way, you could have your test in your CI, but if you found a problem in production, you could run the same test by just referencing the trace ID from production.

Reply:
@mathnogueira that would mean that we'll need a way to create tests without trigger information and a way of test creation only without running. In the case of k6 the idea is to have a static number of tests (created using the tracetest extension) and N number of runs based on the load tests.